tests/integration/README.md

```diff
@@ -0,0 +1,23 @@
+# Integration tests for the Cachito
+
+This directory stores the integration tests for Cachito.
+
+## Configuration
+
+Input data for tests should be configured in a file `test_env_vars.yaml`. The file should be placed in top-level directory of this repository.
+
+See `tests/integration/example_test_env_vars.yaml` for a complete list of configuration options and examples.
+
+## Running tests
+
+Tests can be triggered from the top-level directory of this repository with:
+
+    tox -e integration
+
+The integration environment is not part of the default `tox` envlist.
+
+`REQUESTS_CA_BUNDLE` is passed in `tox.ini` for the `integration`
```

Comment:
I think we want to say "can be passed" here because we don't necessarily have to do this. It depends on the Cachito instance you're running against.

BTW, I always set this variable in my `.bashrc` to prevent headaches. Is that not sufficient here?

Reply:
You can set it in .bashrc too and it will work but I think it's good to mention this here.

Reply:
Changed that to "can be passed"